compiler/parser/src/parser/context.rs

```diff
@@ -257,4 +258,19 @@ impl<'a, N: Network> ParserContext<'a, N> {
     pub(super) fn peek_is_left_par(&self) -> bool {
         matches!(self.token.token, Token::LeftParen)
     }
+
+    /// Error on identifiers that are longer than SnarkVM allows.
+    pub(crate) fn check_identifier(&self, identifier: &Identifier) {
+        const FIELD_CAPACITY_BITS: usize = 250usize;
```

Comment:
You can use [this](https://github.com/AleoNet/snarkVM/blob/4eb83d7f7276514baa9c44b920155750cfe855e7/console/types/field/src/lib.rs#L52) constant to get the number of data bits in a `Field`